core/modules/simpletest/tests/batch.test

```diff
@@ -287,8 +287,11 @@ class BatchPageTestCase extends BackdropWebTestCase {
   function testBatchProgressPageTheme() {
     // Make sure that the page which starts the batch (an administrative page)
     // is using a different theme than would normally be used by the batch API.
-    config_set('system.core', 'theme_default', 'bartik');
-    config_set('system.core', 'admin_theme', 'seven');
+    config('system.core')
```

Comment:
Can we switch this to the syntax where every action completes on a single line. Like this:

```php
$config = config('system.core');
$config->set('admin_theme_same_as_default', 0);
$config->set('theme_default', 'bartik');
$config->set('admin_theme', 'seven');
$config->save();
```

It's both more intuitive and will be consistent with the rest of core.

Reply:
Done 👍